znai-docs/znai/flow/lookup-paths.md

```diff
@@ -15,6 +15,17 @@ When you refer to a file using plugins like
 ../module/src/main/java
 ```
 
+# Class Path Lookup
+
+Znai is written using Java and can access resources in class path. 
+Using class path lookup you can include snippets from projects deployed to, for example, Maven Central.
+
+:include-file: maven-class-path.xml {highlight: "classifier"}
+
+    :include-java: org/junit/Assert.java {entry: "assertArrayEquals"}
+
+:include-java: org/junit/Assert.java {entry: "assertArrayEquals"}
```

Comment:
How can I see how this renders?

Reply:
I don’t build docs during PRs yet. You will have to checkout and run build and then mvn:preview in the docs module

Reply:
![image](https://user-images.githubusercontent.com/11657860/80933933-c56a2c00-8d93-11ea-83fa-214ef4d630df.png)